src/components/FlowRunTimelineSubFlowRunDetails.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <template v-if="subFlowRun">
+    <page-heading :crumbs="crumbs" size="sm" class="mb-2">
+      <template #actions>
+        <slot />
+      </template>
+    </page-heading>
+    <div class="timeline-sub-flow-details__content">
```

Comment:
This class needs to be updated for the component (also inconsistent sub flow usage)